src/main/java/pl/vm/javaguild/designpatterns/pattern/creational/abstractfactory/model/enums/Producer.java

```diff
@@ -0,0 +1,5 @@
+package pl.vm.javaguild.designpatterns.pattern.creational.abstractfactory.model.enums;
```

Comment:
osobiscie nie mam problemow z folderami "Enums" , ale znam architektow ktorzy tego nie toleruja - na przyszlosc mozesz umieszcza je od razu w folderze Model.
Logika jest taka, ze skoro mamy folder Enums do enum'ow to powinnismy miec foldery Interfaces do interface etc.

Reply:
Racja, dzięki!